fuel-gson/src/test/kotlin/com/github/kittinunf/fuel/FuelGsonTest.kt

```diff
@@ -30,6 +38,33 @@ private data class IssueInfo(
     fun specialMethod() = "$id: $title"
 }
 
+private sealed class IssueType {
+    object Bug : IssueType()
+    object Feature : IssueType()
+
+    companion object : JsonSerializer<IssueType>, JsonDeserializer<IssueType> {
+        override fun serialize(src: IssueType, typeOfSrc: Type, context: JsonSerializationContext): JsonElement =
+            when (src) {
+                is Bug -> JsonPrimitive("bug")
+                is Feature -> JsonPrimitive("feature")
+            }
+
+        override fun deserialize(json: JsonElement, typeOfT: Type?, context: JsonDeserializationContext?): IssueType =
+            if (json.isJsonPrimitive() && (json as JsonPrimitive).isString) {
+                when (json.asString) {
+                    "bug" -> Bug
+                    "feature" -> Feature
+                    else -> throw Error("Not a bug or a feature, what is it?")
+                }
+            } else {
+                throw Error("String expected")
+            }
+    }
+}
+
+private typealias IssueTypeList = List<IssueType>
+
+
```

Comment:
[super nits] Can you please remove an extra empty line 🙏 ?

Reply:
Sorry, forgot to run ktlint right before push, fixed

Reply:
No worries ❤️ (_I am not sure why our CI didn't detect this though)

Reply:
reviewdog only run when there is a fail on Travis's PR.